Create ui-c8y-1023-14-92-fixed-issue-with-the-wrong-event-alarm-being-displayed-10907-graft.md (#4301)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-92-fixed-issue-with-the-wrong-event-alarm-being-displayed-10907-graft.md

---
date: ""

> title: fixed issue with wrong event or alarm text being displayed in data graph or data explorer chart.

Reviewer: ```suggestion
title: Data explorer and data point graph display correct event and alarm texts 
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65902
version: 1023.14.92
---

> When viewing events or alarms in the data explorer or data graph chart, there was an issue where incorrect events or alarms text could be displayed to users, potentially causing confusion when monitoring device data or troubleshooting issues. This has been corrected, and the chart now properly displays the correct event or alarm text corresponding to the selected item.

Reviewer: ```suggestion
Previously, when viewing events or alarms in the data explorer or data point graph, incorrect events or alarms text could be displayed to users, potentially causing confusion when monitoring device data or troubleshooting issues. This issue has been fixed, and the chart now properly displays the correct event or alarm text corresponding to the selected item.
```
